src/hotspot/os_cpu/linux_aarch64/vm_version_linux_aarch64.cpp

```diff
@@ -167,3 +167,31 @@ void VM_Version::get_os_cpu_info() {
     fclose(f);
   }
 }
+
+void VM_Version::get_compatible_board(char *buf, int buflen) {
+  const char *aarch64_label = "AArch64";
```

Comment:
VM_Version is not inherited, and platform-specific functions are declared in os_linux and os_windows. So I think it is difficult to declare shared (default) function for this purpose.
It is the best if we declare shared function, and override it like a virtual function. But it seems to be difficult. Do you have any idea?

Reply:
Probably we can assume this function to optionally return a board name. IMHO ideally an empty string should a valid output. Now the output from this function is the prefix and the features string is the suffix of the complete string CPU description. What if we start with the constant `AArch64` prefix in the shared CPU code, append output from this function, and then append the features string?

Reply:
Do you mean `VM_Version::get_compatible_board()` should return `NULL` instead of `AArch64` to add it as a prefix at the caller?
I don't want to do so because we should return board name as possible like a x86.

In Windows AArch64, it seems to have SoC name in registry. Now I do not have Windows AArch64, so I just set "AArch64" for it, but I hope someone work for it in future.
https://stackoverflow.com/questions/60588765/how-to-get-cpu-brand-information-in-arm64

Reply:
Yes, I meant `NULL` or just `""`. Got it, you mean "AArch64" to be a placeholder. But description now starts with `AArch64` prefix, probably worth to keep it. Although it's fine in the current implementation.

Reply:
Ok, I added "AArch64" as a prefix in CPU description in new commit.
We can see it as following:

```
jdk.CPUInformation {
  startTime = 11:51:44.147
  cpu = "AArch64"
  description = "AArch64 raspberrypi,4-model-b brcm,bcm2711 0x41:0x0:0xd08:3, simd, crc"
  sockets = 4
  cores = 4
  hwThreads = 4
}```